src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonContainsTagPredicate;
+
+/**
+ * Lists persons that match all tags specified by the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists persons that match all tags specified\n"
```

Comment:
"persons" could be changed to "classmates" to be more specific to Mass Linkers. 🤔